### ISSUE TITLE
crash on import

### ISSUE THREAD

**Issue by kavemang** [advplyr/audiobookshelf#1473]
### Describe the issue
I already have books and podcasts, its been running fine until just today

```
[2023-02-04 01:20:03] INFO: [Server] Socket Connected L7yZb2P8mpAfCy75AAAF
/server/libs/njodb/index.js:103
        throw error;

TypeError: Cannot convert undefined or null to object

    at Function.values (<anonymous>)

    at ApiRouter.handleUpload (/server/controllers/MiscController.js:22:24)

    at Layer.handle [as handle_request] (/node_modules/express/lib/router/layer.js:95:5)

    at next (/node_modules/express/lib/router/route.js:144:13)

    at Route.dispatch (/node_modules/express/lib/router/route.js:114:3)

    at Layer.handle [as handle_request] (/node_modules/express/lib/router/layer.js:95:5)

    at /node_modules/express/lib/router/index.js:284:15

    at Function.process_params (/node_modules/express/lib/router/index.js:346:12)

    at next (/node_modules/express/lib/router/index.js:280:10)

    at expressInit (/node_modules/express/lib/middleware/init.js:40:5)

```
### Steps to reproduce the issue

1. import folder with mp3s or m4b
2. there will be a burst of traffic for a second or two and then the app crashes, alternatively sometimes there is no traffic to the server at all, the upload hangs and the server crashes
3. container shows exited after this and must be restarted or started

### Audiobookshelf version

2.2.14

### How are you running audiobookshelf?

Docker

**Comment by HubGon** [comment]
I have this problem too. Dont know if it is a coincidence, but i observed it always with audiobooks that have more than 50 audiofiles.

Used version: 2.2.14 with ppa installation

**Comment by advplyr** [comment]
Fixed in [v2.2.15](https://github.com/advplyr/audiobookshelf/releases/tag/v2.2.15)

I updated it so it won't cause a crash, but there may be something more here.
